.eslintrc

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
@d3viant0ne Do you know `.eslintrc` without extension is deprecated?
http://eslint.org/docs/user-guide/configuring#configuration-file-formats